RadixWallet/Features/TransactionReviewFeature/SubmitTransaction/SubmitTransaction.swift

```diff
@@ -180,4 +187,8 @@ extension SubmitTransaction.State.TXStatus {
 		case .notYetSubmitted, .submitting, .submitted, .committedSuccessfully: false
 		}
 	}
+
+	var isFinal: Bool {
+		!isInProgress
```

Comment:
hmm this says that `temporarilyRejected` is final, which sounds like an oxymoron to me. Perhaps not crucial but might be a minor thing you wanna check. I think David Edey is a good person to ask about the meaning of "temporarilyRejected" status, we have not worked to much with it in the past I think (As in Wallet does not use that state a lot).

Reply:
this ended up being removed since we no longer force refresh on final status updates, but inside the AccountLockerClient after awaiting for TX.

Anyway, good to know about this `temporarilyRejected` status to take it into consideration next time.